src/backend/gl/src/window/egl.rs

```diff
@@ -285,10 +298,23 @@ impl hal::Instance<crate::Backend> for Instance {
 
     fn enumerate_adapters(&self) -> Vec<hal::adapter::Adapter<crate::Backend>> {
         let inner = self.inner.lock();
-        inner
-            .egl
-            .make_current(inner.display, None, None, Some(inner.context))
-            .unwrap();
+        if let Some(pbuffer) = inner.pbuffer {
+            inner
```

Comment:
do we actually need this `if`? couldn't we just pass `make_current(inner.display, inner.pbuffer, inner.pbuffer, ..)` right away?

Reply:
Removed `if` and checked on the emulator and device. Everything is working fine.